src/main/java/com/cbs/java_essential/return13/homework/lesson3/ClassRoom.java

```diff
@@ -0,0 +1,119 @@
+package com.cbs.java_essential.return13.homework.lesson3;
+
+//        Задание 2
+//       Используя IntelliJ IDEA, создайте проект.
+//       Требуется: Создать класс, представляющий учебный класс ClassRoom.
+//       Создайте класс ученик Pupil. В теле класса создайте методы void study(), void read(), void write(), void relax().
+//       Создайте 3 производных класса ExcelentPupil, GoodPupil, BadPupil от класса базового класса Pupil и переопределите каждый из методов, в зависимости от успеваемости ученика.
+//       Конструктор класса ClassRoom принимает аргументы типа Pupil, класс должен состоять из 4 учеников. Предусмотрите возможность того, что пользователь может передать 2 или 3 аргумента.
+//       Выведите информацию о том, как все ученики экземпляра класса ClassRoom умеют учиться, читать, писать, отдыхать.
+
+class ClassRoom {
+    ClassRoom(Pupil pupil) {
+        pupil.read();
+        pupil.relax();
+        pupil.study();
+        pupil.write();
+
+    }
+
+    public static void main(String[] args) {
+        ClassRoom pupil = new ClassRoom(new ExcellentPupil());
```

Comment:
Будет работать, но лучше поменяйте следующим образом
```suggestion
        ClassRoom pupil = new ExcellentPupil();
```